libcaliptra/inc/caliptra_types.h

```diff
@@ -102,6 +102,7 @@ struct caliptra_ecdsa_verify_req {
     uint8_t pub_key_y[48];
     uint8_t signature_r[48];
     uint8_t signature_s[48];
+    uint8_t hash[48];
 };
 
 struct caliptra_lms_verify_req {
```

Comment:
nit: Could you change the names of these structures to `*_verify_v2_req`. Makes it a little more clear that the new command is being called.